src/components/barcode/Barcode.tsx

```diff
@@ -20,7 +21,10 @@ const Barcode = (props: BarcodeProps) => {
 
   return (
     <>
-      <div className="barcode-container">
+      <div
+        className={`barcode-container ${
+          className !== undefined ? className : ''
```

Comment:
```suggestion
          className ? className : ''
```
I think this check is unnecessary